plugin/default-plugins/playground/frontend/components/controls/ObjectControl/index.js

```diff
@@ -49,14 +51,14 @@ const ObjectControl = ({ label, propTypeData, value, onUpdate, isNested }) => {
 };
 
 ObjectControl.randomValue = (propTypeData) => {
-  const canBeNull = true;
-  const canBeUndefined = true;
+  const canReallyBeNull = !propTypeData.required;
+  const canReallyBeUndefined = !propTypeData.required;
   const normalizedPropsWithControls = mapValues(propTypeData.value, (prop) => {
     prop.control = getControl(prop); // eslint-disable-line no-param-reassign
     return prop;
   });
   const value = randomValues(normalizedPropsWithControls);
-  return valueOrNullOrUndefined(value, canBeNull, canBeUndefined);
+  return valueOrNullOrUndefined(value, canReallyBeNull, canReallyBeUndefined);
```

Comment:
Oops, that's an artifact from an earlier idea, this should simply be `canBeNull` and `canBeUndefined`. Will fix, good catch!